index.html

```diff
@@ -251,6 +251,9 @@ <h2>Gap between MiniApps and PWA</h2>
                     </tr>
                   </tbody>
                   </table>
+                <p>
+                  Another table in the annex summarizes the <a href="components-comparison.html">differences between MiniApp components and Web Components</a>.
```

Comment:
"Web Components" may not be the right term here, as it usually refers to custom elements. Maybe just "HTML"?

Reply:
Right. I've changed the term to **standard HTML**.

Reply:
Changed here: c41398961aa698e402d5396e318ee5743791060d